examples/query/topic_info_query.py

```
@@ -1,54 +1,64 @@
     TopicCreateTransaction,
 )
 
 
-def create_topic(client, operator_key):
+def create_topic(client):
     """Create a new topic"""
     print("\nSTEP 1: Creating a Topic...")
+
+    operator_key = client.operator_private_key
+    if operator_key is None:
+        raise ValueError("Operator private key not set in environment")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)